nextflow.config

```diff
@@ -62,13 +87,17 @@ profiles {
   }
   singularity {
     singularity.enabled = true
+    singularity.autoMounts = true
   }
   slurm {
     includeConfig 'conf/slurm.config'
   }
-  test {
-    includeConfig 'conf/test.config'
-  }
+  test_illumina { includeConfig 'conf/test_illumina.config' }
+}
+
+conda {
```

Comment:
@nhhaidee I think we should leave the default behaviour for Conda and if the use really wants to `useMamba` they can specify it in a custom config file. I'm not sure if Conda will first install Mamba then use it to install the process Conda envs. Also these should be under the `conda` profile above rather than as a top-level block.